js/templates/settings.html

```diff
@@ -1145,10 +1145,16 @@ <h3 class="padding15 margin0 fontWeight500"><%= polyglot.t('Advanced') %></h3>
               </div>
 
               <div class="flexCol-9 borderRight0 custCol-border">
-                <div class="fieldItem">
-                  <input type="radio" class="fieldItem" id="fancyStylesYes" name="notFancy" value="true" /><label for="fancyStylesYes" class="radioLabel"><%= polyglot.t('Yes') %></label>
-                  <input type="radio" class="fieldItem" id="fancyStylesNo" name="notFancy" value="false" checked/><label for="fancyStylesNo" class="radioLabel"><%= polyglot.t('No') %></label>
-                  <div class="marginLeft10 textSize11px txt-fade pull-right"><%= polyglot.t('NoFancyStylesWarning') %></div>
+                <div class="fieldItem flexRow">
```

Comment:
The fieldItem and flexRow classes shouldn't be used together, I'll remove the flexRow, it's not needed here.